File: integrations/aws-strands/error-code-corpus/typescript/code_through_module_constant.ts
/** A code reached through a module-level constant. */
import { EventType, type BaseEvent } from "@ag-ui/core";

const MODULE_CODE = "THROUGH_MODULE_CONSTANT";

export function emit(): BaseEvent {
  return {
    type: EventType.RUN_ERROR,
    message: "Resolved through a module constant.",
    code: MODULE_CODE,
  };
}

File: integrations/aws-strands/error-code-corpus/python/code_through_module_constant.py
"""A code reached through a module-level constant."""

from ag_ui.core import EventType, RunErrorEvent

MODULE_CODE = "THROUGH_MODULE_CONSTANT"


def emit() -> RunErrorEvent:
    return RunErrorEvent(
        type=EventType.RUN_ERROR,
        message="Resolved through a module constant.",
        code=MODULE_CODE,
    )
